.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: java
+jdk:
+    - openjdk7
+    - oraclejdk7
```

Comment:
Why 7? Can't we use 8?

Reply:
we are maintaining compiler version 1.7 since the connector is used with older product versions also
https://github.com/wso2-extensions/identity-outbound-auth-x509/blob/master/pom.xml#L229